packages/vue/src/components/molecules/SfSearchBar/SfSearchBar.vue

```diff
@@ -14,21 +14,26 @@
     />
     <!-- @slot -->
     <slot name="icon">
-      <span v-if="icon" class="sf-search-bar__icon">
-        <SfIcon :color="icon.color" :size="icon.size" icon="search" />
-      </span>
+      <SfButton
+        class="sf-search-bar__button sf-button--pure"
+        @click="$emit('click', value)"
+      >
+        <span v-if="icon" class="sf-search-bar__icon">
+          <SfIcon :color="icon.color" :size="icon.size" icon="search" />
+        </span>
+      </SfButton>
     </slot>
   </div>
 </template>
 <script>
 import SfIcon from "../../atoms/SfIcon/SfIcon.vue";
-import { focus } from "../../../utilities/directives";
```

Comment:
I'm not sure this will work with adding focus as directives later,
I have tried to add and compile it into Quasar Framework and I have the following error:
"focus is not defined"